Codeinwp/neve
# Various issues

### Summary

- Removed the full page template for new users. Old users can still use the already created pages.
- Customizer Top Bar panel moved to Header.
- Standard fallback menu added for the primary menu location.
- Author gravatar added to the meta.

### Will affect visual aspect of the product

YES

### Screenshots 
<img width="961" alt="neve-gravatar-in-meta" src="https://user-images.githubusercontent.com/26867648/48272318-fe4b6e00-e446-11e8-8a60-46c706a1e3f8.png">

### Test instructions


- Delete the theme mods. 
- Check if primary menu shows all pages from the site.
- Check the Top Bar panel in the customizer.
- Check post page and posts on Blog and Archive pages. The author gravatar should be before the post author name.
